### PR TITLE
created_scheduled_dashboards_doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/dashboards-new/scheduled-report.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+1. Navigate to the dashboard for which you want to create a schedule.
+1. Select or enter a time range for the dashboard.
+1. If the dashboard supports template variables, select values, if desired.
+1. Choose **Create Scheduled Report** from the more options menu in the upper right hand corner of the dashboard.
+
+    <img src={useBaseUrl('img/dashboards-new/scheduled_reports/create_scheduled_report.png')} alt="create_scheduled_report" width="230"/>
+
+1. The **Schedule Report** popup appears.
+
+    **Formart:** Choose PDF or PNG.
```
Comment (stacykor) [inline docs/dashboards-new/scheduled-report.md:28]:
```suggestion
    **Format.** Choose PDF or PNG.
```

File: docs/dashboards-new/scheduled-report.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    **Frequency:** Choose Daily, Weekly, or Monthly.
```
Comment (stacykor) [inline docs/dashboards-new/scheduled-report.md:30]:
```suggestion
    **Frequency.** Choose Daily, Weekly, or Monthly.
```

File: docs/dashboards-new/scheduled-report.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    **Every:** The options depend on the Frequency you selected.
+    * For Daily: You can choose Everyday, Weekdays, or Weekends.
```
Comment (stacykor) [inline docs/dashboards-new/scheduled-report.md:33]:
```suggestion
    *  **Daily.** Choose Every day, Weekdays, or Weekends.
```

File: docs/dashboards-new/scheduled-report.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    * For Weekly: You can choose a particular day of the week.
```
Comment (stacykor) [inline docs/dashboards-new/scheduled-report.md:34]:
```suggestion
    * **Weekly.** Choose a particular day of the week.
```

File: docs/dashboards-new/scheduled-report.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    * For Monthly: You can choose which day of which week of the month you want the report to be sent.
```
Comment (stacykor) [inline docs/dashboards-new/scheduled-report.md:35]:
```suggestion
    * **Monthly.** Choose which day of which week of the month you want the report to be sent.
```

File: docs/dashboards-new/scheduled-report.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Update a scheduled report
+
+To update the existing scheduled dashboard email, do the following:
+
+1. Navigate to the dashboard, and choose **Update Scheduled Report** from the more options menu in the upper right corner of the page. 
+
+1. Make desired changes, and click **Update** to save them. 
+
+<img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report.png')} alt="update_scheduled_report" width="450"/>
```
Comment (kimsauce) [inline docs/dashboards-new/scheduled-report.md:63]: @JV0812 , please close all gaps between sequential steps, per our style.

Assuming "more options" is a UI element, is this titlecased or sentence cased?

```suggestion
1. Navigate to the dashboard, then choose **Update Scheduled Report** from the more options menu in the upper right corner of the page. 
1. Make desired changes and click **Update** to save them. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report.png')} alt="update_scheduled_report" width="450"/>
```

File: docs/dashboards-new/scheduled-report.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+    **Message.** Enter the body of the email.
+
+    <img src={useBaseUrl('img/dashboards-new/scheduled_reports/scheduled_report.png')} alt="scheduled_report>" width="450"/>
+
+:::note
+1. You can schedule only one export per dashboard.
+
+1. Alterting the queries or template variables will not update the scheduled report.
+:::
```
Comment (kimsauce) [inline docs/dashboards-new/scheduled-report.md:53]: Remove numbering; these aren't sequential steps

```suggestion
:::note
*You can schedule only one export per dashboard.
* Altering the queries or template variables will not update the scheduled report.
:::
```

File: docs/dashboards-new/scheduled-report.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+id: scheduled-report
+title: Scheduled Report
+sidebar_label: Scheduled Report
+description: Learn how to create, update, delete and export a scheduled dashboard report.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
```
Comment (kimsauce) [inline docs/dashboards-new/scheduled-report.md:7]:
```suggestion
---

import useBaseUrl from '@docusaurus/useBaseUrl';
```

File: docs/dashboards-new/scheduled-report.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+Scheduled dashboard email helps you to access the dashboard without logging into the Sumo Logic platform, which allows you to distribute the dashboard to any people in the organization who do not have access to the platform enabling better information sharing. Also, it is very useful for compliance management as it automatically captures and archives the state of the dashboard.
+
+## Schedule a dashboard email
+
+To schedule a dashboard email, do the following:
+
+1. Navigate to the dashboard for which you want to create a schedule.
+1. Select or enter a time range for the dashboard.
+1. If the dashboard supports template variables, select values, if desired.
+1. Choose **Create Scheduled Report** from the more options menu in the upper right hand corner of the dashboard.
```
Comment (kimsauce) [inline docs/dashboards-new/scheduled-report.md:22]: Is "more options" in the UI titlecased/sentence-case??

**Review by kimsauce — CHANGES_REQUESTED** [review]
Style, formatting, spelling

**Review by kimsauce — APPROVED** [review]
@JV0812 applied all fixes

**Pull request merged**